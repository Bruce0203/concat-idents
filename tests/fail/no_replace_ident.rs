concat_idents::concat_idents!( = no, _replace {
    fn fn_name() {
        /* there must be a replace char */
    }
});

fn main() {}
