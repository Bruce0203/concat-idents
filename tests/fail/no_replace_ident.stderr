error: expected identifier
 --> $DIR/no_replace_ident.rs:1:32
  |
1 | concat_idents::concat_idents!( = no, _replace {
  |                                ^
